recipes/invoke/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: invoke
+  version: {{ version }}
+
+source:  
+  git_url: https://github.com/pyinvoke/invoke.git
+  git_tag:  {{ version }}
+
+build:
+  number: 0
+  script: python setup.py install
+  entry_points:
+    - invoke = invoke.main:program.run
+    - inv = invoke.main:program.run
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - invoke
+    - invoke.parser
+    - invoke.vendor
+    - invoke.vendor.fluidity
+    - invoke.vendor.lexicon
+    - invoke.vendor.yaml2    # [py2k]
+    - invoke.vendor.yaml3    # [py3k]
+
+  commands:
+    - invoke --help
+    - inv --help
+
+about:
+  home: http://docs.pyinvoke.org
+  license: BSD License
```

Comment:
I think I just copied it from Pypi, which isn't more specific, nor is his setup.py